teamB/jeongyunah/myapp/app.js

```diff
@@ -0,0 +1,55 @@
+
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const logger = require('morgan');
+const app = express();
+const { DataSource } = require('typeorm')
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+appDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  })
+
+app.use(logger('combined'));
+
```

Comment:
request의 JSON data를 가져오기 위해서는 다음과 같은 middleware를 추가해 주어야 합니다.!!
```suggestion
app.use(express.json());
```